components/eam/src/chemistry/utils/prescribed_volcaero.F90

```diff
@@ -212,11 +212,6 @@ subroutine prescribed_volcaero_init(is_cmip6_volc)
     is_cmip6_volc = .false.
     if (trim(adjustl(file_type))== 'VOLC_CMIP6') then
        is_cmip6_volc = .true.
-#if (defined MODAL_AERO_5MODE)
-     write(iulog,*)'kzm_rescribed_volcaero_mam5 '
-     is_cmip6_volc = .false.
-#endif
-
```

Comment:
@keziming 
I removed that part because I found that this can be easily turned off just by setting
 
prescribed_volcaero_file = ‘’
prescribed_volcaero_datapath = ‘’

in the namelist.

It would be better than hardwiring is_cmip6_volc = .false. in prescribed_volcaero.F90